openlibrary/templates/lib/history.html

```diff
@@ -74,7 +74,7 @@ <h2>Wikipedia citation</h2>
                     $if v.author:
                         $ author_link = v.author.render_link(cls="truncate")
                     $elif v.ip and v.ip != "127.0.0.1":
-                        $ author_link = '<a rel="nofollow" href="/recentchanges?ip=%s" title="%s">%s</a>' % (v.ip, _('an anonymous user'), v.ip)
+                        $ author_link = '<a rel="nofollow" href="/recentchanges?ip=%s">%s</a>' % (v.ip, _('an anonymous user'), v.ip)
```

Comment:
This specific one should be left as-is (the `%s` sets the `title` to be a different string).

```suggestion
                        $ author_link = '<a rel="nofollow" href="/recentchanges?ip=%s" title="%s">%s</a>' % (v.ip, _('an anonymous user'), v.ip)
```